dll/win32/comctl32/propsheet.c

```diff
@@ -3254,6 +3254,9 @@ static BOOL PROPSHEET_DoCommand(HWND hwnd, WORD wID)
 	    if (wID == IDOK)
 		{
                     PropSheetInfo* psInfo = GetPropW(hwnd, PropSheetInfoStr);
+#ifdef __REACTOS__
```

Comment:
Since code is now both in Wine and in ReactOS, there's no need for the ifdef.

Reply:
Well, see discussion at https://github.com/reactos/reactos/pull/7940#discussion_r2061640790.